docs/troubleshooting/reindex-elasticsearch.md

```diff
@@ -24,8 +23,138 @@ This will point Cumulus to the index in Elasticsearch that will be used when ret
 
 The reindex operation will take the data from one index and copy it into another index. The reindex operation can be found [here](https://nasa.github.io/cumulus-api/#reindex)
 
+### Reindex status
+
+Reindexing is a long-running operation. The [reindex-status](https://nasa.github.io/cumulus-api/#reindex-status) endpoint can be used to monitor the progress of the operation.
+
 ## Index from database
 
 If you want to just grab the data straight from the database you can perform an [Index from Database Operation](https://nasa.github.io/cumulus-api/#index-from-database). After the data is indexed from the database, a  [Change Index operation](https://nasa.github.io/cumulus-api/#change-index) will need
 to be performed to ensure Cumulus is pointing to the right index. It is **strongly recommended** to turn off
 workflow rules when performing this operation so any data ingested to the database is not lost.
+
+## Validate reindex
+
+To validate the reindex, use the [reindex-status](https://nasa.github.io/cumulus-api/#reindex-status) endpoint. The doc count can be used to verify that the reindex was successful. In the below example the reindex from `cumulus-2020-11-3` to `cumulus-2021-3-4` was not fully successful as they show different doc counts.
+
+```json
+"indices": {
+  "cumulus-2020-11-3": {
+    "primaries": {
+      "docs": {
+        "count": 21096512,
+        "deleted": 176895
+      }
+    },
+    "total": {
+      "docs": {
+        "count": 21096512,
+        "deleted": 176895
+      }
+    }
+  },
+  "cumulus-2021-3-4": {
+    "primaries": {
+      "docs": {
+        "count": 715949,
+        "deleted": 140191
+      }
+    },
+    "total": {
+      "docs": {
+        "count": 715949,
+        "deleted": 140191
+      }
+    }
+  }
+}
+```
+
+To further drill down into what is missing, login to the Kibana instance (found in the Elasticsearch section of the AWS console) and run the following command replacing `<index>` with your index name.
```

Comment:
Super small nit: "log in" instead of "login".

Reply:
👍